runtime/context.go

```diff
@@ -114,18 +115,17 @@ func GetEndpointRequestHeadersFromCtx(ctx context.Context) map[string]string {
 }
 
 // withRequestUUID returns a context with request uuid.
-func withRequestUUID(ctx context.Context, reqUUID uuid.UUID) context.Context {
+func withRequestUUID(ctx context.Context, reqUUID string) context.Context {
```

Comment:
why make this a string instead of a UUID?

Reply:
This is used for log field, which will be `zap.String`, keeping it `uuid.UUID` means useless conversion between UUID and string.